### ISSUE TITLE
Which models expect which audio formats

### ISSUE THREAD

**Issue by kjwill555** [alphacep/vosk-api#276]
Hello,

It'd be very helpful if you could add a column to [the models table](https://alphacephei.com/vosk/models) which indicates the audio format that each model expects.

The Python usage example on the [installation page](https://alphacephei.com/vosk/install) says that the audio should be 16 kHz, 16 bit; however, I know from experience that the English Aspire model expects 8 kHz, 16 bit, so this rule of thumb is misleading.

Basically, I would like to avoid resampling, since I can set the audio format coming out of my microphone beforehand.

Thanks for all you've done!

**Comment by nshmyrev** [comment]
In general we target universal models - i.e. you always feed whatever sample rate you have and you get best accuracy possible. We are just not there yet.

Model description has wideband/narrowband usualy. Narrowband supports 8khz, others 16khz and above. 

There is no need to resample, kaldi resamples internally.

**Comment by kjwill555** [comment]
@nshmyrev Thanks for pointing out the narrowband/wideband mentions, for some reason I didn't notice those at first.

Sorry, I should rephrase: I know Kaldi is capable of resampling, but I imagine that increases processing time. I am performing live online decoding for a voice command system, so processing speed is my main priority. Is resampling fast enough that I shouldn't even worry about it?

**Comment by nshmyrev** [comment]
You should not worry about resampling, it is pretty fast. You only need to care if you send data over the network.

**Comment by kjwill555** [comment]
Ok, good to know. Thanks for your help!
